HelpMyStreetFE/HelpMyStreetFE/Controllers/RegistrationController.cs

```diff
@@ -156,12 +175,19 @@ public async Task<ActionResult> StepThree(CancellationToken cancellationToken)
                 return Redirect(correctPage);
             }
             var registrationFormVariant = await GetRegistrationJourney(user.ID, cancellationToken);
-            var supportActivities = await _groupService.GetSupportActivitesForRegistrationForm(registrationFormVariant);
+            var supportActivities = (await _groupService.GetSupportActivitesForRegistrationForm(registrationFormVariant)).OrderBy(x => x.DisplayOrder).ToList();
```

Comment:
This could be made more efficient by removing `.ToList()` and going straight to `.Select(x => ......`